webpay/urls.py

```diff
@@ -10,16 +12,18 @@
 import jingo.monkey
 jingo.monkey.patch()
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
     (r'^mozpay/auth/', include('webpay.auth.urls')),
     (r'^mozpay/bango/', include('webpay.bango.urls')),
     (r'^mozpay/provider/', include('webpay.provider.urls')),
     (r'^mozpay/services/', include('webpay.services.urls')),
-    (r'^mozpay/', include('webpay.pay.urls'))
+    url(r'^mozpay/$', index, name='index'),
 )
 
 if settings.SPA_ENABLE_URLS:
```

Comment:
These conditionals can all be killed now too. Unless you're saving that for later?